Flow.Launcher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -44,6 +44,7 @@ public SettingWindowViewModel(Updater updater, IPortable portable)
                         break;
                 }
             };
+            ShortCuts = new ObservableCollection<CustomShortcutModel>(Settings.CustomShortcuts.Union(Settings.BuiltinShortcuts));
```

Comment:
I kindly don't like displaying them together in one control....Let's move them to separate control.

Reply:
Also build in shortcut has a different replace policy (no prefix `@` is needed), so shall them share the same name (shortcut)?

Though, I really appreciate your work to help improve the Extensibility. Thanks

Reply:
I separated builtin and custom shortcuts. What's your idea about displaying bulit in shortcuts? Putting them in a list similar to the existing one?

Reply:
![Image](https://user-images.githubusercontent.com/10308169/196024645-9a25fc73-5b4a-4d61-832e-d0edb2c76369.png)

Like this?

Reply:
Yeah that shall be good

Reply:
any other changes needed?